tests/karma/unit/controllers/contacts-content.js

```diff
@@ -92,9 +111,7 @@ describe('ContactsContentCtrl', function() {
       stubFetchChildren(childrenArray);
       createController().getSetupPromiseForTesting()
         .then(function() {
-          assert(scope.setSelected.called, 'setSelected was called');
-          var selected = scope.setSelected.getCall(0).args[0];
-          assertions(selected);
+          assertions(scope.selected);
```

Comment:
The test will fail anyway, but I'll add that to be more obvious why it's failing.